src/AlgebraicGeometry/Schemes/SpecOpen/Objects/Methods.jl

```diff
@@ -187,7 +187,7 @@ function Base.show(io::IO, U::SpecOpen, show_coord::Bool = true)
   if isdefined(U, :name) 
     print(io, name(U))
   elseif get(io, :supercompact, false)
-    print(io, "Scheme")
+    print(io, "Zariski open in affine scheme")
```

Comment:
This is missing a noun ("Zariski open SOMETHING")

Also in the `else` it is `Complement to V(...)`